src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -248,6 +248,18 @@ public partial class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.Reso
             HelpMessage = "Set this flag to true to get a boost on the performance target of the disk deployed, see here on the respective performance target. This flag can only be set on disk creation time and cannot be disabled after enabled.")]
         public bool? PerformancePlus { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Required if createOption is CopyFromSanSnapshot. this is the ARM id of the source elastic san volume snapshot.")]
```

Comment:
Capital This

Reply:
fix with https://github.com/Azure/azure-powershell/pull/23136